container/test/services/container.service.spec.ts

```diff
@@ -68,7 +68,7 @@ describe('getContainerManager messageListener', () => {
     gtcSpy.mockRestore();
   });
 
-  it('test get context message', () => {    
+  it('test get context message', () => {

```

Comment:
The unit test I think is a good addition, but I think we can also add a small Cypress test
Our `iframe/iframeContainer.html` already produces a "cookies" alert automatically when you first open the example. apparently we left it there somehow:
![Screenshot 2024-09-27 at 18 26 50](https://github.com/user-attachments/assets/a85ab097-aa3f-488e-8932-69b0d8f706c0)

You can just check if that alert is present or not to make a test for it or whatever works easiest. 
I tested with your code and this alert doesnt appear when I set **skip-cookie-check** to **true** as expected, but I had to manually get the local script **src**= **"./luigi-client.js"** that I copied manually from **client/public**: 
https://github.com/SAP/luigi/blob/bd332884f4d0362d0a800440c0ec517974b03d17/container/test-app/iframe/microfrontend.html#L7